tianshou/policy/modelfree/a2c.py

```diff
@@ -97,9 +97,10 @@ def forward(self, batch: Batch,
     def learn(self, batch: Batch, batch_size: int, repeat: int,
               **kwargs) -> Dict[str, List[float]]:
         self._batch = batch_size
-        r = batch.returns
+        r = batch.returns[:, None]
```

Comment:
I don't like this notation. Rather use `np.newaxis` instead of None.

Reply:
None is supported for both np and torch

Reply:
I know. But it is very ugly and confusing. I have never seen anybody using it directly in real project.

Reply:
okay